R/readIEA_CCUS.R

```diff
@@ -16,55 +16,90 @@ readIEA_CCUS <- function(subtype) {
   # project types filter applied to source
   projectTypes <- c("Full chain", "Storage", "T&S")
 
-  data <- readxl::read_excel("IEA CCUS Projects Database 2024.xlsx",
+  data <- readxl::read_excel("IEA CCUS Projects Database 2025.xlsx",
     sheet = "CCUS Projects Database"
   ) %>%
     select(
       "project" = "Project name",
-      "country" = "Country",
+      "country" = "Country or economy",
       "type" = "Project type",
       "start" = "Operation",
-      "end" = "Suspension/decommissioning",
+      "end" = "Suspension/decommissioning/cancellation",
       "status" = "Project Status",
       "value" = "Estimated capacity by IEA (Mt CO2/yr)"
     ) %>%
     # remove entries without announced capacities and not matching project type
     filter(!is.na(.data$value), .data$type %in% projectTypes)
 
-  # manually assign shared project(s) to one (currently: first) country
-  data[data$project == "EU2NSEA", "country"] <- "Norway"
-  data[data$country == "Japan-unknown", "country"] <- "Japan"
-
-  # correct typo in IEA Data
+  # correct mistakes in IEA Data
   data[data$country == "Lybia", "country"] <- "Libya"
+  data[data$country == "Island", "country"] <- "Iceland"
```

Comment:
Just a side note: in principle, `madrat::toolCountry2isocode` (you use iz in the convert function) should be able to handle these via the `mapping` argument.